pkg/landscaper/execution/reconcile.go

```diff
@@ -57,11 +57,8 @@ func (o *Operation) Reconcile(ctx context.Context) error {
 				o.exec.Status.Conditions = lsv1alpha1helper.MergeConditions(o.exec.Status.Conditions,
 					lsv1alpha1helper.UpdatedCondition(cond, lsv1alpha1.ConditionFalse,
 						"DeployItemFailed", fmt.Sprintf("DeployItem %s (%s) has not been picked up by a Deployer", item.Info.Name, item.DeployItem.Name)))
-				return lserrors.NewError(
-					"DeployItemReconcile",
-					"DeployItemFailed",
-					fmt.Sprintf("DeployItem %s (%s) has not been picked up by a Deployer", item.Info.Name, item.DeployItem.Name),
-				)
+				o.exec.Status.LastError = lserrors.UpdatedError(o.exec.Status.LastError, "DeployItemReconcile", "DeployItemFailed", fmt.Sprintf("DeployItem %s (%s) has not been picked up by a Deployer", item.Info.Name, item.DeployItem.Name))
```

Comment:
why has this been changed?

Reply:
Because I overestimated the size of your monitor :-P
Will redo this change.